conf/messages

```diff
@@ -359,9 +359,9 @@ upscan.fileTooLarge = The selected file must be smaller than 6MB. Remove the fil
 upscan.fileEmpty = The selected file is empty. Remove the file and try again.
 upscan.fileHasVirus = The selected file contains a virus. Remove the file and try again.
 upscan.unableToUpload = The selected file could not be uploaded. Remove the file and try again.
-upscan.fileUploaded = {0} {1} has been uploaded
-upscan.fileUploading = Uploading {0} {1}
-upscan.fileRemoved = {0} {1} has been removed
+upscan.fileUploaded = File {0} {1} has been uploaded
```

Comment:
This reads as 'File file 1 <name> has been uploaded', removing the 'File' part of this message will rectify it